### DISCUSSION TITLE
support for custom welcome and seeya sounds

### DISCUSSION THREAD

**Discussion by iwa** [ppy/osu#14493] [Ideas] [answered]
...as it is on stable

after some tests it doesn't seem to be implemented yet, is this intentional or just not added?

**Comment by peppy** [comment, marked as answer]
We don't really support skins outside of gameplay yet, so you could say it's just "not implemented yet".

**Reply by iwa** [reply]
alrighty, thanks for the answer!
if i may ask, will it be possible in the future to completely change the appearance of the menus as on stable or it's not something planned?

**Reply by peppy** [reply]
yes
